packages/app/src/app/pages/UserNameSelection/index.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { useEffect } from 'react';
```

Comment:
Maybe we should rename this page to SignUp or something because we don't select the username anymore.

Reply:
done :)